percona-xtradb-cluster-8.0-backup/recovery-azure.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+LIB_PATH='/usr/lib/pxc'
+. ${LIB_PATH}/check-version.sh
+. ${LIB_PATH}/vault.sh
+
+# temporary fix for PXB-2784
+XBCLOUD_ARGS='--curl-retriable-errors=7'
+
+if [ -n "$VERIFY_TLS" ] && [[ $VERIFY_TLS == "false" ]]; then
+	XBCLOUD_ARGS="${XBCLOUD_ARGS} --insecure"
+fi
+
+rm -rf /datadir/*
+tmp=$(mktemp --directory /datadir/pxc_sst_XXXX)
+xbcloud get "${XBCLOUD_ARGS}" "$BACKUP_PATH.sst_info" --storage=azure --parallel=10 | xbstream -x -C "${tmp}" --parallel="$(grep -c processor /proc/cpuinfo)"
+xbcloud get "${XBCLOUD_ARGS}" "$BACKUP_PATH" --storage=azure --parallel=10 | xbstream --decompress -x -C "${tmp}" --parallel="$(grep -c processor /proc/cpuinfo)"
+
+set +o xtrace
+transition_key=$(vault_get "${tmp}/sst_info")
+if [[ -n $transition_key && $transition_key != null ]]; then
+	transition_option="--transition-key=$transition_key"
```

Comment:
Could we also use the same for recover-s3 ? Or use one file for both s3 and azure recover?